scripts/exrtma_run_maxtbg.ksh

```diff
@@ -0,0 +1,243 @@
+#!/bin/bash
```

Comment:
Is there a reason these files are named *.ksh, even though they are bash shells?